pom.xml

```diff
@@ -22,7 +22,7 @@
         <maven.compiler.target>${java.version}</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <information.model.version>4.0.0</information.model.version>
-      	<multipart.message.processor.version>1.0.5-SNAPSHOT</multipart.message.processor.version>
+      	<multipart.message.processor.version>1.0.6-SNAPSHOT</multipart.message.processor.version>
```

Comment:
No need to bump version of MMP lib, no changes there and that one is below wss (no dependency in mmp for wss lib, but other way round)